cmdb/interface/rest_api/__init__.py

```diff
@@ -91,6 +92,7 @@ def register_blueprints(app):
     from cmdb.interface.rest_api.framework_routes.object_link_routes import links_blueprint
 
     app.register_multi_blueprint(auth_blueprint, multi_prefix=['/auth'])
+    app.register_multi_blueprint(date_blueprint, multi_prefix=['/date'])
```

Comment:
Isn't this a settings path? I thought Date is created as Configuration. Then the path should also be accessible via Date. The whole routes are actually obsolete, because they are already implemented under settings routes.